### PR TITLE
updates to the Keen doc

### PR DESCRIPTION
**Pull request by ksigler7**
**Description of the change**: Updates to the new keen.io doc so style is SendGrid flavored md, and some general grammar and style issues
**Reason for the change**: So it actually works in our doc!
**Link to original source**:
https://sendgrid.com/docs/Integrate/Tutorials/analytics_with_keen_io.html
https://sendgrid.com/docs/Classroom/Track/Collecting_Data/a_guide_for_tracking_data_with_keen_io.html

### CODE REVIEW

File: source/Classroom/Track/Collecting_Data/a_guide_for_tracking_data_with_keen_io.md [hunk 1]
```diff
@@ -1,49 +1,45 @@
...
-```
-https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v2?api_key=KEEN_API_WRITE_KEY
-
- ```
-
-In the settings, select the events you want to post to Keen IO (why not all of them?). To do this, check "Select All" under the "Select Actions", or select just the ones you'd like to send to Keen IO.
+1. First click on <a href="http://app.sendgrid.com/settings/mail_settings">Mail Settings</a> within Settings in the left column. 
+2. Next, locate and activate the Event Notification Setting.
+   ![SendGrid- Mail Settings]({{root_url}}/images/keen/sendgrid_mailsettings_eventnotification.png)
+3. Enter the Keen HTTP POST URL, exampled below, as the endpoint where SendGrid will send event notifications. You can find your `KEEN_PROJECT_ID` and `KEEN_API_WRITE_KEY` listed in the [Access tab](https://keen.io/projects) of your Keen project.
```
Comment (Whatthefoxsays) [inline source/Classroom/Track/Collecting_Data/a_guide_for_tracking_data_with_keen_io.md]: Exampled? 

Maybe just "as shown below".

File: source/Classroom/Track/Collecting_Data/a_guide_for_tracking_data_with_keen_io.md [hunk 2]
```diff
@@ -1,49 +1,45 @@
...
+    ```https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v3?api_key=KEEN_API_WRITE_KEY```
+    If you're using Version 1 or 2 of the SendGrid Event API, use this as your Post Event URL instead:
+    ```https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v2?api_key=KEEN_API_WRITE_KEY```
+4. In the settings, select the events you want to post to Keen IO (why not all of them?). To do this, check "Select All" under the "Select Actions", or select just the ones you'd like to send to Keen IO.
```
Comment (Whatthefoxsays) [inline source/Classroom/Track/Collecting_Data/a_guide_for_tracking_data_with_keen_io.md]: The quotes don't really match our style guide. We should decide how to handle Field names/ UI text like this. My suggestion would be:

To do this, check **Select All** under *Select Actions*,

File: source/Classroom/Track/Collecting_Data/a_guide_for_tracking_data_with_keen_io.md [hunk 3]
```diff
@@ -1,49 +1,45 @@
...
 
 That’s it! Now, as your emails make their way through SendGrid, all of the event information will be posted to Keen IO. A new Event Collection, aptly named "SendGrid Email Events", will be created within your project.
 
-
-**Step 3: Send a test email!**
+{% anchor h4 %}
+Step 3: Send a test email!
+{% endanchor h4 %}
 
 To test this out, you can click on the “Test Your Integration” button in the Event Notification settings inside of SendGrid, which should create a set of sample email events from SendGrid into your Keen IO Project.
```
Comment (Whatthefoxsays) [inline source/Classroom/Track/Collecting_Data/a_guide_for_tracking_data_with_keen_io.md]: --
To test this out, click **Test Your Integration** in the SendGrid Event Notification settings.  This creates a set of sample email events from SendGrid into your Keen IO Project.

File: source/Integrate/Tutorials/analytics_with_keen_io.md [hunk 4]
```diff
@@ -9,64 +9,65 @@ layout: page
...
 
+{% anchor h4 %}
+Step 2: Activate SendGrid's Event Notification App
+{% endanchor h4 %}
 
-**Step 2: Activate SendGrid's Event Notification App**
-
-First click on <a href="http://app.sendgrid.com/settings/mail_settings">Mail Settings</a> within Settings in the left column. Next, locate and activate the Event Notification Setting.
-
+1. First click on [Mail Settings](http://app.sendgrid.com/settings/mail_settings) within Settings in the left column. Next, locate and activate the Event Notification Setting.
```
Comment (Whatthefoxsays) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: Click **Mail Settings**.

File: source/Integrate/Tutorials/analytics_with_keen_io.md [hunk 5]
```diff
@@ -9,64 +9,65 @@ layout: page
...
-
-If you're using Version 1 or 2 of the SendGrid Event API, use this as your Post Event URL instead:
-
-```
-https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v2?api_key=KEEN_API_WRITE_KEY
-
- ```
-
-In the settings, select the events you want to post to Keen IO (why not all of them?). To do this, check "Select All" under the "Select Actions", or select just the ones you'd like to send to Keen IO.
+2. Enter the Keen HTTP POST URL, exampled below, as the endpoint where SendGrid will send event notifications. You can find your `KEEN_PROJECT_ID` and `KEEN_API_WRITE_KEY` listed in the [Access tab](https://keen.io/projects) of your Keen project.
```
Comment (Whatthefoxsays) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: Exampled? Is that a word? hahha

File: source/Integrate/Tutorials/analytics_with_keen_io.md [hunk 6]
```diff
@@ -9,64 +9,65 @@ layout: page
...
+    ```https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v3?api_key=KEEN_API_WRITE_KEY```
+    If you're using Version 1 or 2 of the SendGrid Event API, use this as your Post Event URL instead:
+    ```https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v2?api_key=KEEN_API_WRITE_KEY```
+3. In the settings, select the events you want to post to Keen IO (why not all of them?). To do this, check "Select All" under the "Select Actions", or select just the ones you'd like to send to Keen IO.
```
Comment (Whatthefoxsays) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: To do this, check **Select All** from the *Select Actions* menu.

File: source/Integrate/Tutorials/analytics_with_keen_io.md [hunk 7]
```diff
@@ -9,64 +9,65 @@ layout: page
...
 That’s it! Now, as your emails make their way through SendGrid, all of the event information will be posted to Keen IO. A new Event Collection, aptly named "SendGrid Email Events", will be created within your project.
 
 
-**Step 3: Send a test email!**
+{% anchor h4 %}
+Step 3: Send a test email!
+{% endanchor h4 %}
 
-To test this out, you can click on the “Test Your Integration” button in the Event Notification settings inside of SendGrid, which should create a set of sample email events from SendGrid into your Keen IO Project.
+To test this out, you can click on the “Test Your Integration” button in the Event Notification settings inside of SendGrid, which creates a set of sample email events from SendGrid into your Keen IO Project.
```
Comment (Whatthefoxsays) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: To test this out, click **Test Your Integration** in the SendGrid Event Notification settings.  This creates a set of sample email events from SendGrid into your Keen IO Project.

File: source/Integrate/Tutorials/analytics_with_keen_io.md [hunk 8]
```diff
@@ -9,64 +9,65 @@ layout: page
...
 Your SendGrid email data should now be populated in Keen IO. To check, head over to your Keen IO project page. Check out the "Streams" tab, select "SendGrid Email Events" from the drop-down list, and then you can see your Last 10 Events streamed. This should show at least one of our delivered email events, or more if you went to your inbox and opened it!
 
-## <a name="-analyze-your-sendgrid-data"></a>Analyze Your SendGrid Data
+{% anchor h3 %}
+Analyzing Your SendGrid Data
+{% endanchor h3 %}
 
-**Step 1: Log into your Keen account, select your Project**
+{% anchor h4 %}
+Step 1: Log into your Keen account, select your Project
```
Comment (Whatthefoxsays) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: 1. Log into you Keen account, and select your Project.

File: source/Integrate/Tutorials/analytics_with_keen_io.md [hunk 9]
```diff
@@ -9,64 +9,65 @@ layout: page
...
+{% endanchor h4 %}
 
 Open the “Streams” tab, and select the `SendGrid Email Events` to check out your data collection.
 
 ![Keen Streams Manager - SendGrid Email Events]({{root_url}}/images/keen/sendgrid_stream.png)
 
-
-**Step 2: Run some basic queries**
+{% anchor h4 %}
+Step 2: Run some basic queries
```
Comment (Whatthefoxsays) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: 2. Run some basic queries.

File: source/Integrate/Tutorials/analytics_with_keen_io.md [hunk 10]
```diff
@@ -75,14 +76,20 @@ Enter `count` as the “Analysis Type”, select `SendGrid Email Events` as your
...
+{% endanchor h4 %}
 
 ![Save Your Query]({{root_url}}/images/keen/save_query.png)
 
+{% anchor h4 %}
+Step 2:  Add your graph to a dashboard
+{% endanchor h4 %}
 
-**Step 2:  Add your graph to a dashboard.**
 Select “Create Dashboard”, name your dashboard, and add the query we saved to the first tile. Then click “Save”.
```
Comment (Whatthefoxsays) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: Remove quotes and make the UI elements bold.

File: source/Integrate/Tutorials/analytics_with_keen_io.md [hunk 11]
```diff
@@ -75,14 +76,20 @@ Enter `count` as the “Analysis Type”, select `SendGrid Email Events` as your
...
 
 
-## Save Your Query & Make Your First Dashboard
+{% anchor h3 %}
+Saving Your Query & Making Your First Dashboard
+{% endanchor h3 %}
 
-**Step 1: Let’s save your first query**
+{% anchor h4 %}
+Step 1: Save your first query
```
Comment (Whatthefoxsays) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: remove "step 1"

File: source/Integrate/Tutorials/analytics_with_keen_io.md [hunk 12]
```diff
@@ -75,14 +76,20 @@ Enter `count` as the “Analysis Type”, select `SendGrid Email Events` as your
...
+{% endanchor h4 %}
 
 ![Save Your Query]({{root_url}}/images/keen/save_query.png)
 
+{% anchor h4 %}
+Step 2:  Add your graph to a dashboard
```
Comment (Whatthefoxsays) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: Remove "step 2"

File: source/Integrate/Tutorials/analytics_with_keen_io.md [hunk 13]
```diff
@@ -92,10 +99,14 @@ Congratulations! You’ve created your first dashboard!
...
+Doing Drilldowns and Reporting on Your Data
+{% endanchor h3 %}
 
-## <a name="-doing-drilldowns-and-reporting-on-your-data"></a>Doing Drilldowns and Reporting on Your Data
+{% anchor h4 %}
+Select_Unique
+{% endanchor h4 %}
 
-### Select_Unique
 `Select_Unique` allows you extract many properties from a single field and filter on a specific segment of users based on any number of fields such as campaigns, clicks, and companyID.
```
Comment (Whatthefoxsays) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: Should Select_Unique be in quotes or a different font? I'm stuck on how we should differentiate our UI elements/field names when there's not an action.

File: source/Integrate/Tutorials/analytics_with_keen_io.md (hunk 4 above)
Comment (ksigler7) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: I verified these steps, and this is correct.

File: source/Integrate/Tutorials/analytics_with_keen_io.md (hunk 5 above)
Comment (ksigler7) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: haha, it is now! Jk, I fixed it.

File: source/Integrate/Tutorials/analytics_with_keen_io.md (hunk 8 above)
Comment (ksigler7) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: Re to next few comments - I actually did this for the first round, and it didn't look very good. There's too much going on in each section for it not to break md lists, so I'm going to leave it for now.

File: source/Integrate/Tutorials/analytics_with_keen_io.md (hunk 13 above)
Comment (ksigler7) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: I made it in `code` syntax

File: source/Integrate/Tutorials/analytics_with_keen_io.md [hunk 14]
```diff
@@ -136,7 +152,10 @@ Here’s an example funnel:
...
 
-## <a name="-embedding-sendgrid-analytics-into-your-app"></a>Embedding SendGrid Analytics Into Your App
+{% anchor h4 %}
+Embedding SendGrid Analytics Into Your App
+{% endanchor h4 %}
+
 Just like SendGrid, Keen IO is 100% powered by APIs. This means you can embed rich analytics anywhere you can write code. Here’s a link to our [Native Analytics Guide](https://keen.io/guides/native-analytics/) that will give you step-by-step instructions on how to embed your SendGrid analytics into your product.
 
-*That concludes our guide for the day. Hope you have some fun with data and happy hacking!*
+*Hope you have some fun with data and happy hacking!*
```
Comment (Whatthefoxsays) [inline source/Integrate/Tutorials/analytics_with_keen_io.md]: Have fun with data and happy hacking!

**Pull request merged**